java-plugin/src/main/java/fr/greencodeinitiative/java/JavaRulesDefinition.java

```diff
@@ -39,14 +39,14 @@
 public class JavaRulesDefinition implements RulesDefinition {
 
     // don't change that because the path is hard coded in CheckVerifier
-    private static final String RESOURCE_BASE_PATH = "fr/cnumr/l10n/java/rules/java";
+    private static final String RESOURCE_BASE_PATH = "fr/greencodeinitiative/l10n/java/rules/java";
 
 
     // Add the rule keys of the rules which need to be considered as template-rules
     private static final Set<String> RULE_TEMPLATES_KEY = Collections.emptySet();
     public static final String NAME = "Collectif Conception Numérique Responsable ";
```

Comment:
Shall we keep this name?

Reply:
no, you're right. correction done (and in other files also)